src/main/app/case/to-api-format.ts

```diff
@@ -325,6 +325,8 @@ const languagePreferenceYesNoOrNull = (value: LanguagePreference | undefined) =>
   return value === LanguagePreference.Welsh ? YesOrNo.YES : YesOrNo.NO;
 };
 
+const addressConverter = (address: (string | undefined)[]) => (address.some(Boolean) ? address.join('\n') : '');
+
```

Comment:
why do we need to do this, we can just do a for-loop in the template to display it correctly

Reply:
this is the to-api-format so it's not about how we display the field it's how it's sent to the backend. 

I've updated it so it just sends an empty string as oppose to "\n\n\n\n\n" which is how it was being sent before if it was empty, and therefore incorrectly showing up in xui (as per the ticket)

Reply:
so is the backend expecting it in that format, with the newlines?

Reply:
it is yeah if it's not completely empty, if it's empty then it should be a blank string